lora/lora.c

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+*
+* FILE: 
+* 		lora.c
+*
+* DESCRIPTION: 
+* 		Contains API functions for transmating and receiving from the board's
+*       built-in LoRa module.
+*
+*******************************************************************************/
+
+#include "stm32h7xx_hal.h"
+/*------------------------------------------------------------------------------
+ Standard Includes  
+------------------------------------------------------------------------------*/
+#include <stdbool.h>
+
+
+/*------------------------------------------------------------------------------
+ MCU Pins 
+------------------------------------------------------------------------------*/
+#if   defined( FLIGHT_COMPUTER   )
+	#include "sdr_pin_defines_A0002.h"
+#elif defined( GROUND_STATION    )
+    #include "sdr_pin_defines_A0005.h"
+#endif
+
+/*------------------------------------------------------------------------------
+ Helper functions for various pin functions on the LoRa modem.
+------------------------------------------------------------------------------*/
+void lora_write_cs_pin( GPIO_PinState pinState ) {
```

Comment:
Pass in the boolean or a readable enum type instead of GPIO_PinState. Try not to pass or refer HAL functions